### PR TITLE
Bring in menu UI dex

### PR DESCRIPTION
**Pull request by ape-swap**

### CODE REVIEW

File: src/widgets/Menu/Menu.tsx [hunk 1]
```diff
@@ -66,11 +66,14 @@ const Menu: React.FC<NavProps> = ({
   toggleTheme,
   langs,
   setLang,
+  toggleNotPossible,
```
Comment (ape-swap) [inline src/widgets/Menu/Menu.tsx:69]: I don't follow what `toggleNotPossible` means.

File: src/widgets/Menu/Menu.tsx [hunk 2]
```diff
@@ -132,6 +139,7 @@ const Menu: React.FC<NavProps> = ({
           isDark={isDark}
           toggleTheme={toggleTheme}
           langs={langs}
+          toggleNotPossible={toggleNotPossible !== undefined && true}
```
Comment (ape-swap) [inline src/widgets/Menu/Menu.tsx:142]: Doesn't `toggleNotPossible={toggleNotPossible}` do the same?

File: src/widgets/Menu/PanelFooter.tsx [hunk 3]
```diff
@@ -115,13 +116,15 @@ const PanelFooter: React.FC<Props> = ({
...
         <Button variant="text" onClick={() => toggleTheme(!isDark)}>
           {/* alignItems center is a Safari fix */}
-          <Flex alignItems="center">
-            <SunIcon color={isDark ? "textDisabled" : "text"} width="24px" />
-            <Text color="textDisabled" mx="4px">
-              /
-            </Text>
-            <MoonIcon color={isDark ? "text" : "textDisabled"} width="24px" />
-          </Flex>
+          {toggleNotPossible ? null : (
```
Comment (ape-swap) [inline src/widgets/Menu/PanelFooter.tsx:119]: Maybe `disableTheme` is a better name?

File: src/widgets/Menu/Menu.tsx (hunk 1 above)
Comment (APeelingApe) [reply src/widgets/Menu/Menu.tsx:69]: Toggle not possible is for saying don't have toggle (light/dark).    In DEX we don't allow light mode currently.

File: src/widgets/Menu/Menu.tsx (hunk 2 above)
Comment (APeelingApe) [reply src/widgets/Menu/Menu.tsx:142]: I was getting a typescript error and needed the !== undefined

Related to the intrinsic attributes piece

File: src/widgets/Menu/PanelFooter.tsx (hunk 3 above)
Comment (APeelingApe) [reply src/widgets/Menu/PanelFooter.tsx:119]: Yes it could be that

**Pull request closed without merging**